src/Bitrix24/README.md

```diff
@@ -11,11 +11,10 @@ Please see the [Base Installation Guide](https://socialiteproviders.com/usage/),
 ### Add configuration to `config/services.php`
 
 ```php
-'bitix24' => [
+'bitrix24' => [
       'endpoint' => env('BITRIX24_ENDPOINT_URI'),
       'client_id' => env('BITRIX24_CLIENT_ID'),
       'client_secret' => env('BITRIX24_CLIENT_SECRET'),
-      'redirect' => env('BITRIX24_REDIRECT_URI'),
```

Comment:
Generally in ouath2 you need to pass this as a URL param, as well as setting in the provider's platform. Is this not the case with bitrix?